ext_mongodb.php

```diff
@@ -746,7 +868,7 @@ public function getCode() : string
 
 	public function getScope() : mixed
 	{
-		if ( $this->scope !== NULL )
+		if ( isset( $this->scope) && $this->scope !== NULL )
```

Comment:
Shouldn't `$scope` always be declared? Otherwise, you're liable to create it as a public property at runtime.

If it's declared private, there should be no need for `isset()` here.